### DISCUSSION TITLE
Does standalone version have web frontend?

### DISCUSSION THREAD

**Discussion by jbrepogmailcom** [Koenkk/zigbee2mqtt#15012] [General]
Hello, after a year of use I discovered that Home Assistant version of zigbee2mqtt have web frontend. I use standalone npm version of zigbee2mqtt - does that have web frontend or is there any possibility how to set it up? And how to configure? Thank you.

**Comment by Koenkk** [comment]
All z2m version have the frontend included. See https://www.zigbee2mqtt.io/guide/configuration/frontend.html how to enable.
